docs/distribute/create-or-select-a-distribution-profile.md

```diff
@@ -54,189 +188,176 @@ Under the Auto Send tab in the settings, you can see the testing groups you have
+## Share Binary
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/image (158).png' />
+### Share your application with the test groups manually
 
-###
+When you have your build ready or uploaded your binary file to Appcircle, you can manually send the file to testers to let them download, install on their devices and run the application for testing purposes.
 
-### New version from build module
+Click on the Share with Testers button and select the [testing groups](/distribute/testing-groups) you created previously that need to receive this version of your application. Alternatively, you can enter email addresses of the testers here to send them the application regardless of the testing groups.
 
-With a successful build, a new version of your application will be added to your distribution profile.
+You can also add a message to testers including testing instructions and release notes.
 
-Simply go to _Build Module_ _>_ _Build Configuration_ _>_ _Distribution_ and select a distribution profile you want your build to be sent.
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4163-main14.png' />
 
-:::tip
+You can automate this message using [Release Notes Component](https://github.com/appcircleio/appcircle-release-notes-component/). You can enrich the contents of your release notes with environment variables or Ruby snippets. The following default template will print the branch name, commit hash and commit message.
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-3167-buildoutput.png' />
+```ruby
+Branch: $AC_GIT_BRANCH
+Commit Hash:  <%= ENV['AC_GIT_COMMIT'][0..6] %>
+Commit Message: $AC_COMMIT_MESSAGE
+```
 
-Users have the capability to select their Android build output as .AAB (Android App Bundle) from configuration settings within their Build profile.
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4163-mail.png' />
 
-When the `.AAB` build is sent to the designated Testing Distribution profile, either automatically or manually by uploading the file directly within the Testing Distribution profile, it will be automatically converted to `.APK` format when shared with a Testing Group. This ensures that the receiving tester downloads the shared artifact in `.APK` format.
+:::tip
 
-This conversion capability also applies when sending app versions from a Testing Distribution profile to an [Enterprise App Store](/enterprise-appstore) profile. The shared `.AAB` artifact will be downloaded in `.APK` format from the Enterprise App Store profile.
+Once you share your app versions with your testers, you can see the most recent sharing time on your testing distribution profile card.
 
 :::
 
-:::caution
+:::info
 
-Only signed builds will be distributed. Unsigned builds cannot be distributed.
+If you are using Self Hosted version of Appcircle, instead of the default noreply@appcircle.io, you can use your own business domain for the distribution emails.
```

Comment:
```suggestion
If you are using the Self-Hosted version of Appcircle, you can configure it to use your own business domain for distribution emails instead of the default noreply@appcircle.io.
```